module/zfs/dsl_crypt.c

```diff
@@ -935,9 +949,56 @@ spa_keystore_unload_wkey(const char *dsname)
 	return (ret);
 }
 
+void
+key_mapping_add_ref(dsl_key_mapping_t *km, void *tag)
+{
+	ASSERT3U(refcount_count(&km->km_refcnt), >=, 1);
+	zfs_refcount_add(&km->km_refcnt, tag);
```

Comment:
`zfs_refcount_add()` seems to be a rename of `refcount_add()` - perhaps namespace collision. I assume I can just rename it back for the other platforms - unless it is desired to have the one weird name on illumos as well :)

Reply:
@lundman: maybe related: https://github.com/zfsonlinux/zfs/pull/7963

Reply:
@lundman As soon as #7963 is (hopefully) merged, all the refcount functions will have the zfs_ prefix for consistency. It is planned to upstream those to OpenZFS and other forks as well.

Reply:
I suspected as much - for what it's worth, it passes all the tests here. :)